sample/src/main/res/layout/activity_nested_scroll.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Well, this LinearLayout has only one child. That's something that we need to avoid, isn't it?